en/apispec/openapi.json

```diff
@@ -3057,360 +2530,2865 @@
+        }
+      ],
       "get": {
-        "summary": "Get API version infomation",
+        "summary": "Query Service Quota Usage by customer ID",
+        "tags": [
+          "ServiceQuotas"
+        ],
         "responses": {
           "200": {
             "description": "OK",
-            "headers": {
-              "Qtl-Request-Id": {
-                "schema": {
-                  "type": "string"
-                },
-                "description": "A string automatically generated by the API server to identify each request."
-              }
-            },
             "content": {
               "application/json": {
                 "schema": {
                   "type": "object",
                   "properties": {
-                    "code": {
-                      "$ref": "#/components/schemas/response.code"
-                    },
-                    "status": {
-                      "$ref": "#/components/schemas/response.status"
-                    },
-                    "message": {
-                      "$ref": "#/components/schemas/response.message"
+                    "resourceQuotas": {
+                      "$ref": "#/components/schemas/serviceQuota.resourceQuotas"
                     },
-                    "content": {
+                    "resourceQuotasUsed": {
                       "type": "object",
+                      "description": "Global resource quotas available to a customer.",
                       "properties": {
-                        "version": {
-                          "type": "string",
-                          "description": "The version of the current API build. It’s format is ‘release-{yyyymmdd}’"
-                        },
-                        "artifact": {
-                          "type": "string",
-                          "description": "The artifact of the current build."
-                        },
-                        "name": {
-                          "type": "string",
-                          "description": "The name of the API artifact."
-                        },
-                        "group": {
-                          "type": "string",
-                          "description": "The group name of current API build."
+                        "transports": {
+                          "type": "integer",
+                          "description": "The number of available transports a customer used"
                         },
-                        "time": {
-                          "type": "string",
-                          "description": "The date time when the current version is built."
+                        "trafficVolume": {
+                          "type": "number",
+                          "description": "Unit GB, the traffic volume a customer transported.\nNote: This item shall only be available for self-signup users, as the calculation of the total traffic volume of a customer ever used will be very time consuming. "
```

Comment:
Suggestion:  "Traffic volume in GB used by a customer. "  
I think you can just mention it is only for self-signup users without mentioning the limit on the calculation.